src/pages/DataAccessRequestApplication_new.js

```diff
@@ -285,7 +285,13 @@ const DataAccessRequestApplicationNew = (props) => {
   };
 
   const updateDraftResponse = async (formattedFormData, referenceId) => {
-    return await DAR.updateDarDraft(formattedFormData, referenceId);
+    let darPartialResponse;
+    if(!isNil(referenceId) && !isEmpty(referenceId)) {
+      darPartialResponse = await DAR.updateDarDraft(formattedFormData, referenceId);
+    } else {
+      darPartialResponse = await DAR.postDarDraft(formattedFormData);
```

Comment:
This change currently redirects to the old page, not sure how to get it to redirect to the new page instead.